core/client/gobindclient/client.go

```diff
@@ -107,7 +110,9 @@ func AddKtServer(ktURL string, insecureTLS bool, ktTLSCertPEM []byte, directoryI
 		}
 	}
 
-	client, err := client.NewFromConfig(ktClient, config)
+	client, err := client.NewFromConfig(ktClient, config,
+		func(lv *tclient.LogVerifier) verifier.LogTracker { return tracker.New(lv) },
```

Comment:
Potentially in a future CL, I recommend renaming tracker.New to something like tracker.NewSynchronousHead or equivalent. This is something that may have many different implementations that could all be reasonable given different requirements.